app/views/form-designer/make-your-form-live.html

```diff
@@ -0,0 +1,92 @@
+{% extends "layout-govuk-forms.html" %}
+
+{% set pageTitle = 'Make your form live' %}
+
+{% block pageTitle %}
+  {{ "Error: " if containsErrors }} {{pageTitle}} - {{ data['formTitle'] }} - GOV.UK Forms
+{% endblock %}
+
+{% block beforeContent %}
+  <a class="govuk-back-link" href="create-form">Back to create a form</a>
+{% endblock %}
+
+{% block content %}
+
+{% if containsErrors %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: errorList
+  }) }}
+{% endif %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <span class="govuk-caption-l">{{ data['formTitle'] }}</span>
+    <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
+
+    <p class="govuk-body">
+      When you make your form live you will get a URL for the form. The form will not be indexed by search engines, so people will not be able to find it easily. Contact your GOV.UK publishing team to publish a link to your form on GOV.UK so people can find it. 
+    <p>
+
+    <p class="govuk-body">
+      Make sure you are happy with the form before you make it live. After you have made your form live:
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          completed forms will be sent to [the email address they provided]
+        </li>
+        <li>
+          you will not be able to change the name of the form.
+        </li>
+      </ul>
+    </p>
```

Comment:
I think the bullet list issue is because the paragraph tag is closed down here. I think moving the closing tag to above the `<ul>` should fix it :)